diff --git a/source/java/org/rsna/util/PrintUtil.java b/source/java/org/rsna/util/PrintUtil.java
index d4ae472..7207fbd 100644
--- a/source/java/org/rsna/util/PrintUtil.java
+++ b/source/java/org/rsna/util/PrintUtil.java
@@ -5,7 +5,7 @@
 *  RSNA Public License (http://mirc.rsna.org/rsnapubliclicense.pdf)
 *----------------------------------------------------------------*/

-package org.rsna.ui;
+package org.rsna.util;

 import java.awt.*;
 import javax.swing.*;
